gtfs-netex-test/epip_db_to_xml.py

```diff
@@ -88,7 +88,7 @@ def generator(self) -> Generator | None:
 
         return None
 
-def export_epip_network_offer(database_original, database_target, output_filename: str):
+def main(database_original, database_target, output_filename: str):
```

Comment:
Not what should be changed. Happy to make a main, but this is a generic function.